config/rbac/status-reporter-clusterrole.yaml

```diff
@@ -22,11 +22,11 @@ rules:
       - list
       - update
   - apiGroups:
```

Comment:
pls move this rbac to next existing stanza and just add `dns` and also remove `watch` verb on that stanza.

Reply:
Done